modules/apps/user-associated-data/user-associated-data-web/src/main/resources/META-INF/resources/add_uad_export_processes.jsp

```diff
@@ -60,7 +60,7 @@ renderResponse.setTitle(StringBundler.concat(selectedUser.getFullName(), " - ",
 				}
 				%>
 
-				<clay:management-toolbar-v2
+				<clay:management-toolbar
 					disabled="<%= disableManagementBar %>"
 					itemsTotal="<%= uadApplicationExportDisplayList.size() %>"
 					namespace="<%= liferayPortletResponse.getNamespace() %>"
```

Comment:
We can simply remove the `namespace` attribute here, is is set directly in the Tag.

Reply:
👍